### PR TITLE
Add login page

### PR DESCRIPTION
**Pull request by IrinaSendetskaya**

### CODE REVIEW

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  selector: "app-login",
+  templateUrl: "./login.component.html",
+  styleUrls: ["./login.component.css"]
+})
+export class LoginComponent implements OnInit {
+  constructor() {}
+  user: User = new User();
+
+  addUser() {
+    console.log(this.user);
```
Comment (nastabulak) [inline src/app/login/login.component.ts]: do you really need it?

File: src/app/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  }
+
+  checkUser() {
```
Comment (nastabulak) [inline src/app/login/login.component.ts]: do you check user or its password?))

File: src/app/add-course/add-course.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-add-course',
```
[the remark this answers is not in the document]
Comment (IrinaSendetskaya) [reply src/app/add-course/add-course.component.ts]: changed

File: src/app/add-course/add-course.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  templateUrl: './add-course.component.html',
+  styleUrls: ['./add-course.component.css']
+})
+export class AddCourseComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```
[the remark this answers is not in the document]
Comment (IrinaSendetskaya) [reply src/app/add-course/add-course.component.ts]: changed

File: src/app/courses/courses.component.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.css']
+})
+export class CoursesComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```
[the remark this answers is not in the document]
Comment (IrinaSendetskaya) [reply src/app/courses/courses.component.ts]: changed

File: src/app/edit-course/edit-course.component.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-edit-course',
+  templateUrl: './edit-course.component.html',
+  styleUrls: ['./edit-course.component.css']
+})
+export class EditCourseComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```
[the remark this answers is not in the document]
Comment (IrinaSendetskaya) [reply src/app/edit-course/edit-course.component.ts]: changed

File: src/app/login/login.component.css [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+.alert-danger {
+  color: red;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: row;
+  padding: 10px 10px 10px 10px;
+  margin: 10px 10px 10px 10px;
```
[the remark this answers is not in the document]
Comment (IrinaSendetskaya) [reply src/app/login/login.component.css]: changed

File: src/app/login/login.component.ts (hunk 1 above)
Comment (IrinaSendetskaya) [reply src/app/login/login.component.ts]: changed

File: src/app/login/login.component.ts (hunk 2 above)
Comment (IrinaSendetskaya) [reply src/app/login/login.component.ts]: changed

**Pull request merged**